### PR TITLE
extension package

### LINKED ISSUE #26: extension package
The extension loaders (both `rax_extensions` and `standard_extensions`) attempt to load every module/package inside `__path__`, recursively. Because packages and modules that are not extensions exist, we should move the extensions to their own package.

I vote something like `contrib`, for consistency with cinder.

### PR DESCRIPTION
**Pull request by buhman**
This moves all extensions to the `contrib` package while keeping compatibility with the old `rackspace_cinder_extensions.rax_extensions` loader.

Fixes #26.

### CODE REVIEW

**Comment by thrawn01** [comment]
@corywright This change will require some modification to cinder.conf. 

```
osapi_volume_extension=rackspace_cinder_extensions.rax_admin.Rax_admin
```
Becomes
```
osapi_volume_extension = rackspace_cinder_extensions.contrib.select_extensions
```

**Comment by buhman** [comment]
No, it doesn't.

https://github.rackspace.com/BigData/cbs-salt/blob/master/files/base/common/etc/cinder/cinder.conf#L25

`rackspace_cinder_extensions.rax_extensions` will continue to work after this is merged.

**Comment by thrawn01** [comment]
@buhman ```rackspace_cinder_extensions.rax_extensions``` is deprecated, it is NOT required before we merge this PR, but it does need to be changed. Notice in my earlier comment I didn't say it was *required* I just want @corywright to be aware we are making this change, and give him time to provide feedback if needed.

File: README.md [hunk 1]
```diff
@@ -24,16 +24,11 @@ Now build the `.deb` file:
 
 Add this line to `cinder.conf`:
 
-    osapi_volume_extension = rackspace_cinder_extensions.rax_extensions
+    osapi_volume_extension = rackspace_cinder_extensions.api.contrib.standard_extensions
 
-### Load Extension: os-snapshot-progress
+### Load extensions by name:
 
 Add this line to `cinder.conf`:
```
Comment (corywright) [inline README.md]: Change:

   Add this line to `cinder.conf`:

to:

    Add these lines to `cinder.conf`:

Comment (corywright) [inline README.md]: Change:

    Add this line to `cinder.conf`:

to:

    Add these lines to `cinder.conf`:

**Pull request merged**